app/scripts/snaps/preinstalled-snaps.ts

```diff
@@ -5,8 +5,8 @@ import BitcoinWalletSnap from '@metamask/bitcoin-wallet-snap/dist/preinstalled-s
 ///: END:ONLY_INCLUDE_IF
 
 // The casts here are less than ideal but we expect the SnapController to validate the inputs.
-const PREINSTALLED_SNAPS: readonly PreinstalledSnap[] = Object.freeze([
-  MessageSigningSnap as unknown as PreinstalledSnap,
+const PREINSTALLED_SNAPS = Object.freeze<PreinstalledSnap[]>([
```

Comment:
```suggestion
const PREINSTALLED_SNAPS = Object.freeze<ReadOnlyArray<PreinstalledSnap>>([
```
Can we do this instead since before the type was readonly?

Reply:
This would be redundant since the return type of `Object.freeze<T>` is `readonly T`. By hovering over `PREINSTALLED_SNAPS`, you can verify that its type is `readonly PreinstalledSnap[]` like before.

If we want to be extra careful about validating the type maybe we could add a type annotation or a `satisfies` clause?

```diff
diff --git a/app/scripts/snaps/preinstalled-snaps.ts b/app/scripts/snaps/preinstalled-snaps.ts
index 6be0bb7f35..95b9c54e69 100644
--- a/app/scripts/snaps/preinstalled-snaps.ts
+++ b/app/scripts/snaps/preinstalled-snaps.ts
@@ -10,6 +10,6 @@ const PREINSTALLED_SNAPS = Object.freeze<PreinstalledSnap[]>([
   ///: BEGIN:ONLY_INCLUDE_IF(build-flask)
   BitcoinWalletSnap as unknown as PreinstalledSnap,
   ///: END:ONLY_INCLUDE_IF
-]);
+]) satisfies readonly PreinstalledSnap[];
 
 export default PREINSTALLED_SNAPS;
```

Reply:
Bug report

Reply:
@daniela2000111 Is there a bug related to the changes in `preinstalled-snaps.ts`?

Reply:
Ah yes that's correct! Fine to leave as is :)

Reply:
Curious about this bug report ^